### PR TITLE
make the formatting errors based on go 1.19

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind cleanup

```release-note
NONE
```

Additional Note for reviewer:

The errors in absence of this PR cause tests to fail in https://github.com/kubernetes-csi/csi-release-tools/pull/203

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/kubernetes-csi/external-snapshotter/v6
 
-go 1.17
+go 1.19
```
Comment (pohly) [inline go.mod:3]: Why this version bump?

It causes the unit test to fail (still runs with Go 1.18, which doesn't support `go 1.19`) and shouldn't be necessary (go 1.19 can compile code that only depends on features from `go 1.17`).

**Comment by xing-yang** [comment]
Please don't update anything in the client folder yet.
/hold

File: vendor/github.com/kubernetes-csi/external-snapshotter/client/v6/apis/volumesnapshot/v1/types.go [hunk 2]
```diff
@@ -123,11 +123,11 @@ type VolumeSnapshotSource struct {
 // VolumeSnapshotStatus and VolumeSnapshotContentStatus. Fields in VolumeSnapshotStatus
 // are updated based on fields in VolumeSnapshotContentStatus. They are eventual
 // consistency. These fields are duplicate in both objects due to the following reasons:
-// - Fields in VolumeSnapshotContentStatus can be used for filtering when importing a
-//   volumesnapshot.
-// - VolumsnapshotStatus is used by end users because they cannot see VolumeSnapshotContent.
-// - CSI snapshotter sidecar is light weight as it only watches VolumeSnapshotContent
-//   object, not VolumeSnapshot object.
+//   - Fields in VolumeSnapshotContentStatus can be used for filtering when importing a
```
[the remark this answers is not in the document]
Comment (humblec) [reply vendor/github.com/kubernetes-csi/external-snapshotter/client/v6/apis/volumesnapshot/v1/types.go:126]: I was referrring the update in client go.mod with version bump to kube v1.25 https://github.com/kubernetes-csi/external-snapshotter/pull/758/files#diff-de2c00fe7d49a7bb4d5c225eded980780a7b9dde4ae89e70c7e497cd0e45b15bL9-R6 by lift the dependencies.

**Comment by pohly** [comment]
IMHO all that this PR should do is make the formatting compliant with Go 1.19 and nothing else. Then we can merge https://github.com/kubernetes-csi/csi-release-tools/pull/203.

**Comment by humblec** [comment]
> IMHO all that this PR should do is make the formatting compliant with Go 1.19 and nothing else. Then we can merge [kubernetes-csi/csi-release-tools#203](https://github.com/kubernetes-csi/csi-release-tools/pull/203).

ok.. let me shrink the PR to just do formatting based on 1.19 in that case.

**Comment by humblec** [comment]
@pohly @xing-yang as per the above discussion, I have kept only formatting change here in this PR. ptal.. Thanks..

File: pkg/common-controller/snapshot_controller.go [hunk 3]
```diff
@@ -782,11 +782,12 @@ func (ctrl *csiSnapshotCommonController) storeContentUpdate(content interface{})
 // given event on the snapshot. It saves the status and emits the event only when
 // the status has actually changed from the version saved in API server.
 // Parameters:
-//   snapshot - snapshot to update
-//   setReadyToFalse bool - indicates whether to set the snapshot's ReadyToUse status to false.
-//                          if true, ReadyToUse will be set to false;
-//                          otherwise, ReadyToUse will not be changed.
-//   eventtype, reason, message - event to send, see EventRecorder.Event()
+//
+//	snapshot - snapshot to update
```
Comment (pohly) [inline pkg/common-controller/snapshot_controller.go:786]: I think a bullet list is more appropriate here.

File: pkg/sidecar-controller/snapshot_controller.go [hunk 4]
```diff
@@ -137,8 +137,9 @@ func (ctrl *csiSnapshotSideCarController) checkandUpdateContentStatus(content *c
 // given event on the content. It saves the status and emits the event only when
 // the status has actually changed from the version saved in API server.
 // Parameters:
-//   content - content to update
-//   eventtype, reason, message - event to send, see EventRecorder.Event()
+//
+//	content - content to update
```
Comment (pohly) [inline pkg/sidecar-controller/snapshot_controller.go:141]: And here.

**Review by pohly — CHANGES_REQUESTED** [review]

File: pkg/sidecar-controller/snapshot_controller.go (hunk 4 above)
Comment (humblec) [reply pkg/sidecar-controller/snapshot_controller.go]: done @pohly .. ptal.. thanks.

**Comment by pohly** [comment]
Can you remove the dependency update commit? It is unrelated to this PR.

**Comment by pohly** [comment]
You probably added it because of a Trivy failure, but that's just bad style. I'm really getting annoyed by this security check.

**Comment by humblec** [comment]
> Can you remove the dependency update commit? It is unrelated to this PR.

The  Trivvy  scnner fail here in this PR on lastest run, so clubbed with the same. I can split that to another PR too and mark it as dependency for this PR. 
@pohly  wdyt?

**Comment by humblec** [comment]
> You probably added it because of a Trivy failure, but that's just bad style. I'm really getting annoyed by this security check.

Indeed . Thats why I refreshed with that change. Let me split that up first.

**Comment by humblec** [comment]
The trrivvy failure is addressed here https://github.com/kubernetes-csi/external-snapshotter/pull/759

**Comment by humblec** [comment]
> The trrivvy failure has been addressed here #759

As the trivvy failure has been addressed on master, I have rebased this PR.  This should be good to go as well @xing-yang @pohly

**Review by pohly — APPROVED** [review]
/lgtm

**Pull request merged**